aiosmtpd/smtp.py

```diff
@@ -663,9 +885,8 @@ async def smtp_DATA(self, arg):
             return
         # Remove extraneous carriage returns and de-transparency
         # according to RFC 5321, Section 4.5.2.
-        for i in range(len(data)):
```

Comment:
Whenever you see `range(len(o))`, 99% of the time it's replaceable with `enumerate(o)`.